.travis.yml

```diff
@@ -70,6 +70,24 @@ jobs:
       # Disable pip cache dir temporarily
       install:
        - pip install --no-cache-dir -r requirements-selftests.txt
+    - name: "Ad hoc checks on OS X"
+      os: osx
+      language: c
+      osx_image: xcode10.3
+      compiler: clang
+      addons:
+        homebrew:
+          packages:
+            - python
+          update: false
+      install:
+        - python3 -m pip install -r requirements-selftests.txt
+        - python3 setup.py develop --user
+      script:
+        - python3 -m avocado --help
+        - python3 -m avocado nlist --verbose selftests/unit/* selftests/functional/* selftests/*sh
+        - python3 -m unittest discover -v selftests.unit
```

Comment:
IMO, it is not ideal for the project to have releases or commits with the X red mark (failed). Sometimes we ignore those because we know that it is a "safe to ignore" failures.

So maybe, we could add this using the "allow_failures" in Travis?

All green it is good!

Reply:
Never mind, just saw that it is already there. :)